### ISSUE TITLE
use official java-s2i image

### ISSUE THREAD

**Issue by jorgemoralespou** [redhat-cop/agnosticd#12]
Now that an official s2i-java image has been released, we should switch to that one. Needs to be present on the environment.
https://blog.openshift.com/red-hat-brings-cloud-native-services-to-every-java-workload/

cc @thoraxe

**Comment by thoraxe** [comment]
Cool -- I will try to add that to my fork for future merge. Thanks @jorgemoralespou

**Comment by thoraxe** [comment]
```json
{
    "kind": "ImageStream",
    "apiVersion": "v1",
    "metadata": {
        "name": "redhat-openjdk18-openshift"
    },
    "spec": {
        "dockerImageRepository": "registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift",
        "tags": [
            {
                "name": "1.0",
                "annotations": {
                    "description": "OpenJDK S2I images.",
                    "iconClass": "icon-jboss",
                    "tags": "builder,java,xpaas",
                    "supports":"java:8,xpaas:1.0",
                    "sampleRepo": "https://github.com/jboss-openshift/openshift-quickstarts",
                    "sampleContextDir": "undertow-servlet",
                    "version": "1.0"
                }
            }
        ]
    }
}
```

https://gist.githubusercontent.com/tqvarnst/3ca512b01b7b7c1a1da0532939350e23/raw/3869a54c7dd960965f0e66907cdc3eba6d160cad/openjdk-s2i-imagestream.json

**Comment by thoraxe** [comment]
Installation of the IS is fixed by #21 

Still need to change the lab content but that is pending a PR as well.

**Comment by gmontalvoy** [comment]
Looks like this was fixed at that moment (2017) and it is using openjdk8 at the moment.

Candidate to close @fridim ?
